lib/fastlane/actions/backup_file.rb

```diff
@@ -0,0 +1,31 @@
+module Fastlane
+  module Actions
+    class BackupFileAction < Action
+      def self.run params
+        path = params[:path]
+        FileUtils.cp(path, "#{path}.back", {:preserve => true})
+      end
+
+      def self.description
+        'This action backup your file to "[path].back".'
+      end
+
+      def self.is_supported?(platform)
+        true
+      end
+
+      def self.author
+        "gin0606"
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :path,
+                                       env_name: "",
```

Comment:
https://github.com/fastlane/fastlane_core/blob/master/lib/fastlane_core/configuration/config_item.rb#L18

`env_name` must give String. If drop `env_name` or give `nil` then raise RuntimeError. it say "env_name must be a String" :cry: